challenge_8/java/jdfurlan/src/RandomPointerLinkedList.java

```diff
@@ -0,0 +1,62 @@
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Random;
+/**
+ * 
+ * @author jesse furlan
+ * 
+ * THIS CODE IS INCOMPLETE
```

Comment:
It would be good to have this in the README.md for this challenge. That file gets shown when navigating to the directory in github so the notice would be quite visible to anyone looking.